<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>External editor</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Editor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="externalEditorName">
            <property name="text">
             <string>gimp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="externalEditorSelect">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Image format</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>DPI:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="imageDpi">
            <property name="maximum">
             <number>2400</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Format:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="imageFormat">
            <item>
             <property name="text">
              <string>jpeg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>tiff</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>png</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bmp</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>218</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>226</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Formats</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" rowspan="5">
          <widget class="QListWidget" name="formatsList">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="formatUp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/up.png</normaloff>:/interface/images/bottons/up.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="formatDown">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/down.png</normaloff>:/interface/images/bottons/down.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="formatDel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/list-remove.png</normaloff>:/interface/images/bottons/list-remove.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="formatSave">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/apply.png</normaloff>:/interface/images/bottons/apply.png</iconset>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QToolButton" name="formatAdd">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/list-add.png</normaloff>:/interface/images/bottons/list-add.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="7">
          <widget class="QLineEdit" name="formatName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="formatWidth">
           <property name="maximum">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Height:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="formatHeight">
           <property name="maximum">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Top to Pate:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QSpinBox" name="formatTopToPateDistance">
           <property name="maximum">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Nose to Chin:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QSpinBox" name="formatNoseToChinDistance">
           <property name="maximum">
            <number>1024</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>118</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Matrix</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" rowspan="5">
          <widget class="QListWidget" name="matrixList">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="matrixUp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/up.png</normaloff>:/interface/images/bottons/up.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="matrixDown">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/down.png</normaloff>:/interface/images/bottons/down.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="matrixDel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/list-remove.png</normaloff>:/interface/images/bottons/list-remove.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="matrixSave">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/apply.png</normaloff>:/interface/images/bottons/apply.png</iconset>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QToolButton" name="matrixAdd">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../rc/main.qrc">
             <normaloff>:/interface/images/bottons/list-add.png</normaloff>:/interface/images/bottons/list-add.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="matrixName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="matrixPosDel">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../rc/main.qrc">
               <normaloff>:/interface/images/bottons/list-remove.png</normaloff>:/interface/images/bottons/list-remove.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="matrixPosX">
               <property name="maximum">
                <number>1024</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="matrixPosY">
               <property name="maximum">
                <number>1024</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="1">
            <widget class="QToolButton" name="matrixPosAdd">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../rc/main.qrc">
               <normaloff>:/interface/images/bottons/list-add.png</normaloff>:/interface/images/bottons/list-add.png</iconset>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" rowspan="6">
            <widget class="QListWidget" name="matrixPosList"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Paper</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Width:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="matrixPaperWidth">
                <property name="maximum">
                 <number>1024</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Height:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="matrixPaperHeight">
                <property name="maximum">
                 <number>1024</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Banner</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Text:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="matrixBannerText"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>X:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="matrixBannerX">
                  <property name="maximum">
                   <number>1024</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Y:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="matrixBannerY">
                  <property name="maximum">
                   <number>1024</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Font size:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="matrixBannerFontSize"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="settingsButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/main.qrc"/>
 </resources>
 <connections/>
</ui>
